astropy/table/pprint.py

```diff
@@ -13,19 +13,26 @@
 import numpy as np
 
 from .. import log
-# Note, in numpy <= 1.6, some classes do not properly represent themselves.
 from ..utils.console import Getch, color_print, terminal_size, conf
 from ..utils.data_info import dtype_info_name
 
-if six.PY3:
+__all__ = []
+
+if six.PY2:  # pragma: py2
+    def default_format_func(format_, val):
+        if isinstance(val, bytes):
+            return val.decode('utf-8')
+        else:
+            return text_type(val)
+else:  # pragma: py3
     def default_format_func(format_, val):
         if isinstance(val, bytes):
             return val.decode('utf-8')
         else:
             return str(val)
```

Comment:
I thought about that, but doesn't that make it less efficient, since it has to compare at every call? Doing it this way, the function is set once it determines whether user is running Python 2 or 3.